python/paddle/fluid/contrib/slim/tests/test_mkldnn_int8_quantization_strategy.py

```diff
@@ -0,0 +1,127 @@
+#   copyright (c) 2019 paddlepaddle authors. all rights reserved.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+
+import paddle
+import unittest
+import os
+import tarfile
+import struct
+import numpy as np
+import paddle.fluid as fluid
+from mobilenet import MobileNet
+from paddle.fluid.contrib.slim.core import Compressor
+from paddle.fluid.contrib.slim.graph import GraphWrapper
+from paddle.dataset.common import download
+
+
+class TestInferQuantizeStrategy(unittest.TestCase):

```

Comment:
Why this UT enabled on MAC/windows? It should be on LInux only.

Reply:
understand much thanks!